compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -1200,20 +1243,19 @@ impl<K: DepKind> CurrentDepGraph<K> {
         edges: EdgesVec,
         current_fingerprint: Fingerprint,
     ) -> DepNodeIndex {
```

Comment:
This function should probably be renamed `alloc_new_node` as it no longer does interning.